src/components/TextField.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const TextField = ({label, name, placeholder, value, handleChange, type}) => {
```

Comment:
same name as attribute names may show warnings and cause problems sometimes, you can customise it differently here. eg. , label -> inputLabel